src/app/components/application-areas/application-areas-list/application-areas-list.component.ts

```diff
@@ -61,7 +65,7 @@ export class ApplicationAreasListComponent implements OnInit {
   }
 
   onAddElement(): void {
-    const params: any = {};
+    // On checking the references params has only body set and then used in line 83
```

Comment:
referencing to a line is not helpful in the long run as code may change

Reply:
in addition, I am not sure if the comment supports understanding - the code (as it is no complex code) _should_ be understandable enough such that it is clear what is done.